src/app/client/src/app/modules/workspace/components/published/published.component.spec.ts

```diff
@@ -77,7 +80,7 @@ describe('PublishedComponent', () => {
   // If search api returns more than one published
   it('should call search api and returns result count more than 1', inject([SearchService], (searchService) => {
     spyOn(searchService, 'compositeSearch').and.callFake(() => observableOf(testData.searchSuccessWithCountTwo));
-    component.fetchPublishedContent(9, 1);
+    component.fetchPublishedContent(9, 1, { queryParams: { subject: ['english', 'odia'] }});
```

Comment:
The assertion is missing to check where the call to a method which fetches the content having these params or not

Reply:
I wrote different test cases for  this